pages/app/app-context.tsx

```diff
@@ -8,10 +8,10 @@ import { useHistory, useLocation, useRouteMatch } from 'react-router-dom';
 
 interface AppUrlParams {
   density: Density;
+  direction: string;
```

Comment:
I would change that to a union `"ltr" | "rtl"`

Reply:
All set!